core-lib/Hello.som

```diff
@@ -1,8 +1,27 @@
-class Hello usingPlatform: platform = Value ()(
+class Hello usingPlatform: platform = Value (
+)(
   public main: args = (
-    'Hello World!' println.
-    args from: 2 to: args size do: [ :arg | arg print. ' ' print ].
-    '' println.
-    ^ 0
+    | x2 |
```

Comment:
Please don't change the hello world :)
Feel free to add more tests though, also possibly 'basic interpreter tests' if that makes debugging easier (see the folder and JUnit tests).